### PR TITLE
webapp

### PR DESCRIPTION
**Pull request by strejcik**
Stworzyłem podstronę IDO, komponenty, dodałem je też do storybooka i zrobiłem responsywność

### CODE REVIEW

**Review by awojciak — COMMENTED** [review]
first of all: PR-s should go to dev branch, not master or staging

File: src/pages/PagesRouter.tsx [hunk 1]
```diff
@@ -46,6 +47,7 @@ export const PagesRouter: React.FC = () => {
             path={'/position/:id'}
             render={({ match }) => <SinglePositionPage id={match.params.id} />}
           />
+          <Route path={'/ido'} component={IdoPage} />
```
Comment (awojciak) [inline src/pages/PagesRouter.tsx:50]: it was not necessary to create page, only component and storybook for it were in scope of task

**Review by awojciak — COMMENTED** [review]
there is something wrong with storybook in your repo - I can't run it

File: src/components/Ido/Ido.tsx [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+            </Grid>
+            <Box className={classes.tokenComponentTextContainer}>
+                <Typography className={classes.tokenComponentText}>EST.:{estValue} </Typography>
+                <Typography className={classes.tokenComponentText}>Balance:{balanceValue}</Typography>
+            </Box>
+            <TokenAmountInput/>
+                <Typography className={classes.tokenComponentText}>Deposited Amount:</Typography>
+                <Grid container spacing={0}> 
+                    <Grid item>
+                        <img src="/src/static/png/logo.png" className={classes.invariantLogoIDO}></img>
```
Comment (awojciak) [inline src/components/Ido/Ido.tsx:45]: you should use svg images instead of png

File: src/components/Ido/Ido.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+                    </Grid>
+                    <Grid item>
+                        <Typography className={classes.xUSD}>${xUSD}</Typography>
+                        <Typography className={`${classes.tokenComponentText} ${classes.USDSOLxETxBTC}`}>
+                            {USD}<span className={classes.tab}></span>{SOL} <span className={classes.tab}></span>{xETH} <span className={classes.tab}></span>{xBTC} 
+                        </Typography>
+                    </Grid>
+                </Grid>
+                <ConnectAWalletButton
```
Comment (awojciak) [inline src/components/Ido/Ido.tsx:54]: why won't you just use our animated button? and why is this button doing nothing?

File: src/components/Inputs/TokenAmountInput/TokenAmountInput.tsx [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+    
+    return (
+        <Input
+          //inputRef={}
+          //error={}
+          className={classNames(classes.amountInput, "")}
+          classes={{ input: classes.input }}
+          //style={}
+          type={'text'}
+          //value={}
```
Comment (awojciak) [inline src/components/Inputs/TokenAmountInput/TokenAmountInput.tsx:19]: something should be done with value

**Review by awojciak — COMMENTED** [review]
your component:
image
mockup:
image
differences can be seen easily

**Comment by strejcik** [comment]
I've intentionally put the components without propses, so you could add propses and/or logic. The task was about putting the components in order, make responsiveness and to make a visual representation of mockup. If button / input 'does not work' it's because it lacks of propses which I gave up as I mentioned before.
Figma got me confused, because you can zoom in and out mockups. I didn't know exactly how to get correct sizes- my bad. Meaby if every mockup would have been in *.psd, it would be easier- because you can see font sizes, font names and exact spaces between elements.
Components have been added to storybook and works- checked that.

**Comment by awojciak** [comment]
> I've intentionally put the components without propses, so you could add propses and/or logic. 
You could just set mock data in story for component. Also some inner logic in component should be working, for example: clicking on max button in input should set its value to max available. Look at other stories in our repo.
> Figma got me confused, because you can zoom in and out mockups. I didn't know exactly how to get correct sizes- my bad. 
You can get these by clicking on elements on mockups.

**Pull request closed without merging**